### PR TITLE
fix(gtk): Fix get scan code in GtkKeyboardInputSource

### PR DESCRIPTION
**Pull request by lindexi**
## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Now the GtkKeyboardInputSource will use the [InputHelper.TryConvertKeyCodeToScanCode](https://github.com/unoplatform/uno/blob/292e14b3f933dba83d78363d0729d488908c6a8d/src/Uno.UI/Helpers/InputHelper.cs) to convert the key code to scan code which will always return false. It means the scan code same as the key code.

## What is the new behavior?

The GtkKeyboardInputSource will use the gtk to get the scan code

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

See https://docs.gtk.org/gdk3/method.Event.get_scancode.html

Reference https://github.com/unoplatform/uno/pull/16670

Origin commit : 98c965a794b058541cafe6756c46f021b1017d5c

Cc @ramezgerges

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Gtk/Input/GtkKeyboardInputSource.cs [hunk 1]
```diff
@@ -49,15 +49,15 @@ private void OnKeyPressEvent(EventKey evt)
 				this.Log().Trace($"OnKeyPressEvent: {evt.Key} -> {virtualKey}");
 			}
 
-			InputHelper.TryConvertKeyCodeToScanCode(evt.HardwareKeycode, out var scanCode);
```
Comment (Youssef1313) [inline src/Uno.UI.Runtime.Skia.Gtk/Input/GtkKeyboardInputSource.cs:52]: `TryConvertKeyCodeToScanCode` is probably unused now and can be removed?

Comment (lindexi) [reply src/Uno.UI.Runtime.Skia.Gtk/Input/GtkKeyboardInputSource.cs:52]: @Youssef1313 Yeah. And I will remove it.

**Review by ramezgerges — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
Thanks for your contribution!

**Comment by lindexi** [comment]
Thank you.

**Pull request merged**